content/en/serverless/azure_app_services.md

```diff
@@ -41,11 +42,16 @@ The Datadog extension for Azure App Services provides additional monitoring capa
 
 1. If you haven't already, set up the [Microsoft Azure integration][8] first.
 
-2. The extension only supports Azure App Service Web Apps. **Function Apps are not supported**.
+2. The extension only supports:
 
-<div class="alert alert-warning">To be notified when a private beta is available for Function Apps or additional runtimes, <a href="https://forms.gle/n4nQcxEyLqDBMCDA7">sign up here</a>.</div>
+    - Azure App Service Web Apps
+    - Function Apps hosted on Premium or Standard App Service Plans (beta for extension v1.3.4+). **Note**: Function Apps on consumption plans are not supported.
 
-3. The Datadog .NET APM extension supports the following .NET runtimes in both x64 and x86 architectures when running on Windows instances (AAS does not yet support extensions on Linux). For more details about automatically instrumented libraries, see the [Tracer documentation][9].
+    <div class="alert alert-warning">Support for .NET Function Apps (except on consumption plans) is now in beta for extension v1.3.4+. There are no billing implications for tracing functions during this period.</div>
```

Comment:
This yellow callout basically just repeats the same thing that's in the previous bullet. Let's change the message here a bit:

"There are no billing implications for tracing functions during the beta period. Interested in support for additional resource types or runtimes? [Sign up here][9] to be notified when a beta becomes available."